### ISSUE TITLE
subsequent display of view will add more component

### ISSUE THREAD

**Issue by hainm** [arose/nglview#705]
```
view = nv.demo()
view # 2nd cell
view # 3rd cell
view # 4th cell
view # 5th cell
```

Then the number of components will be
```
2nd cell: 4
3rd cell: 3
4th cell: 2
5th cell: 1
```

ack.

**Comment by hainm** [comment]
that makes sense because when display view again, 'loadFile' is called for all views. :((

**Comment by hainm** [comment]
Also mean that if we call `view.download_image()`, there will be 4 files.

**Comment by hainm** [comment]
fixed in #704
